app/assets/javascripts/preview.js

```diff
@@ -0,0 +1,25 @@
+$(document).on("turbolinks:load", function () {
+  $("#card-preview").click(function(){
+    previewCard();
+    highlight();
+  });
+});
+
+function previewCard() {
+  marked.setOptions({
+    renderer: new marked.Renderer(),
+    gfm: true,
+    tables: true,
+    breaks: true,
+    pedantic: false,
+    sanitize: false,
+    smartLists: true,
+    smartypants: false
+  });
+
+  var text = $("#card-description").val();
```

Comment:
if you wanted to be cute and remove the if statement hinge for pure logic you could write it like this:
$(".preview").html(text && marked(text) ||  "Nothing to preview...")

but maybe that's, what they call in bird culture, a dick move